80s Game/Assets/Scripts/Target.cs

```diff
@@ -44,7 +44,7 @@ void Start()
         // Init component references
         movementControls = GetComponent<KinematicSteer>();
         animControls = GetComponent<AnimationHandler>();
-        collider = GetComponent<CircleCollider2D>();
+        collider = GetComponent<PolygonCollider2D>();
```

Comment:
I'll need to readd this to the code for my PR. A lot of code got migrated during my refactor, just a couple lines tho so nbd